deploy/src/main/resources-filtered/production-kubernetes-prerequisites/jobservice-prerequisites-deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: jobservicedb
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      creationTimestamp: ${maven.build.timestamp}
+      labels:
+        io.kompose.service: jobservicedb
+    spec:
+      containers:
+      - name: jobservicedb
+        image: postgres:9.6
+        env:
+        - name: POSTGRES_PASSWORD
+          value: "${JOB_SERVICE_DB_PASSWORD}"
+        - name: POSTGRES_USER
+          value: "${JOB_SERVICE_DB_USER}"
+        resources: {}
```

Comment:
empty, not mandatory
Remove in all other occurrences, too.

Reply:
Removed